src/main/kotlin/no/nav/familie/klage/behandling/dto/PåklagetVedtakDetaljerMapper.kt

```diff
@@ -9,6 +9,7 @@ fun FagsystemVedtak.tilPåklagetVedtakDetaljer() = PåklagetVedtakDetaljer(
     behandlingstype = this.behandlingstype,
     eksternFagsystemBehandlingId = this.eksternBehandlingId,
     fagsystemType = this.fagsystemType,
+    vedtakType = this.vedtakType,
```

Comment:
Siden vi hels ikke skal ha to verdier i basen så kan det være lurt å:
1) Sjekk om vedtakType er satt -> bruk denne -> Hvis ikke godta fagsystemType